x/auth/types/stdtx.go

```diff
@@ -97,15 +98,56 @@ func (tx StdTx) GetSigners() []sdk.AccAddress {
 // GetMemo returns the memo
 func (tx StdTx) GetMemo() string { return tx.Memo }
 
-// GetSignatures returns the signature of signers who signed the Msg.
 // GetSignatures returns the signature of signers who signed the Msg.
 // CONTRACT: Length returned is same as length of
 // pubkeys returned from MsgKeySigners, and the order
 // matches.
 // CONTRACT: If the signature is missing (ie the Msg is
 // invalid), then the corresponding signature is
 // .Empty().
-func (tx StdTx) GetSignatures() []StdSignature { return tx.Signatures }
+func (tx StdTx) GetSignatures() [][]byte {
```

Comment:
why did the return type change?

Reply:
To satisfy `GetSignatures` method of `SigVerifiableTx` interface. This is necessary for `StdTx` to work with the decorators in `sigverify.go`